### PR TITLE
created The README.md file

### PR DESCRIPTION
**Pull request by ddomb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,142 @@
...
+-->
+
+<!-- PROJECT LOGO -->
+<h1 align="center">GymBot</h>
+<br />
+<p align="center">
+  <h3 align="center">Telegram Gym Bot</h3>
+
+  <p align="center">
+    YOUR_SHORT_DESCRIPTION
```
Comment (AsafSH6) [inline README.md:34]: "Telegram bot to forgroups which want to help each other keep the motivation"

use italic

File: README.md [hunk 2]
```diff
@@ -1,2 +1,142 @@
...
+* []()Python 2.7
+* []()MongoDB
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+## Prerequisites
```
Comment (AsafSH6) [inline README.md:70]: It doesn't really look like sub-bullet of the getting started, is there a way to do it?
maybe use "###"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,142 @@
...
+
+Telegram bot token
+</br>
+
+1.  Download the Telegram app to your phone
+2.  Create telegram bot
+```sh
+https://core.telegram.org/bots#creating-a-new-bot
+```
+> **when finishing to create the bot its important to save his token.**
```
Comment (AsafSH6) [inline README.md:80]: nice

File: README.md [hunk 4]
```diff
@@ -1,2 +1,142 @@
...
+```sh
+git clone https://github.com/AsafSH6/GymBot.git
+```
+2. Enter the project directory
+```sh
+cd ./GymBot
+``` 
+2. Install dependencies
+```sh
+Run pip install -r requirements.txt
```
Comment (AsafSH6) [inline README.md:104]: Remove the word "run" since it is ovious and without its easier to copy the line

File: README.md [hunk 5]
```diff
@@ -1,2 +1,142 @@
-# GymBot
-telegram gym bot
+
```
Comment (AsafSH6) [inline README.md:1]: Please add a few screenshots

File: README.md [hunk 6]
```diff
@@ -1,2 +1,142 @@
...
+    <a href="https://github.com/AsafSH6/GymBot/issues">Request Feature</a>
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [Built With](#built-with)
```
Comment (AsafSH6) [inline README.md:48]: Add section to actually use the bot in telegram
1. Create telegram group with your friends
2. Add GymiousBot to the group
3. Use the bot commands set to start

**Review by AsafSH6 — COMMENTED** [review]
looking good

File: README.md (hunk 5 above)
Comment (ddomb) [reply README.md:1]: image
image
image
this is before the changes you asked for

Comment (ddomb) [reply README.md:1]: image
image
image
this is before the changes you asked for

Comment (ddomb) [reply README.md:1]: I got trouble with taking screenshots, but here is the link for the branch
https://github.com/ddomb/GymBot/tree/feature/Create-Readme

File: README.md (hunk 6 above)
Comment (ddomb) [reply README.md:48]: added at the bottom (Usage)

**Review by AsafSH6 — APPROVED** [review]

**Pull request merged**